modal/_utils/hash_utils.py

```diff
@@ -49,11 +49,18 @@ class UploadHashes:
     sha256_base64: str
 
 
-def get_upload_hashes(data: Union[bytes, BinaryIO]) -> UploadHashes:
+def get_upload_hashes(data: Union[bytes, BinaryIO], sha256_hex: Optional[str] = None) -> UploadHashes:
     md5 = hashlib.md5()
```

Comment:
We could also compute the md5 digest in parallel with the first sha256 digest, on a separate thread. Future work.